samples/mixed/consumer/src/main.rs

```diff
@@ -115,49 +115,22 @@ async fn get_provider_uri(entity_id: &str) -> Result<String, String> {
     let mut client = DigitalTwinClient::connect(IN_VEHICLE_DIGITAL_TWIN_SERVICE_URI)
         .await
         .map_err(|error| format!("{error}"))?;
-    let request = tonic::Request::new(FindByIdRequest { entity_id: String::from(entity_id) });
-    let response = client.find_by_id(request).await.map_err(|error| format!("{error}"))?;
-    let dtdl = response.into_inner().dtdl;
+    let request = tonic::Request::new(FindByIdRequest { id: String::from(entity_id) });
+    let response = client.find_by_id(request).await.map_err(|error| error.to_string())?;
+    let response_inner = response.into_inner();
     debug!("Received the response for the find_by_id request");
+    info!("response_payload: {:?}", response_inner.entity_access_info);
+
+    let provider_uri = match response_inner.entity_access_info {
```

Comment:
consider using expect here similar to a previous suggestion

Reply:
Done.